redisinsight/ui/src/pages/rdi/pipeline/components/dry-run-job-commands/DryRunJobCommands.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react'
+import { useSelector } from 'react-redux'
+import parse from 'html-react-parser'
+import { monaco } from 'react-monaco-editor'
+
+import { CodeBlock } from 'uiSrc/components'
+import { rdiDryRunJobSelector } from 'uiSrc/slices/rdi/dryRun'
+import { MonacoLanguage } from 'uiSrc/constants'
+
+const DryRunJobCommands = () => {
+  const { results } = useSelector(rdiDryRunJobSelector)
+
+  const [commands, setCommands] = useState('')
+
+  useEffect(() => {
+    if (results && results.commands?.status === 'success') {
```

Comment:
<img width="702" alt="Screenshot 2024-01-09 at 2 23 03 PM" src="https://github.com/RedisInsight/RedisInsight/assets/112435081/94c1bcbc-7325-4f87-801d-7a3de9c074a3">

Should `commands` and `status` be added to the `IPipeline` interface?

Reply:
Thank you. I updated interface